static/css/legacy/main.css

```diff
@@ -942,7 +937,7 @@ form.favorite {
     max-width: 90em;
     min-width: 960px;
     position: relative;
-    min-height: 55em;
+    min-height: 768px; /* arbitrary high number to force the footer down */
```

Comment:
mmm?  How would this be affected if the user had a screen height bigger than 768px (and etc)

Reply:
FWIW `55em`, the previous value, is already arbitrary and once computed without messing with the font size would be roughly equivalent to `715px`. I just increased it a bit so that it matches the footer better visually (since that new footer is longer in size).

With that new value, a typical page would be 768+65+1+606 pixels long with the default font size on my machine. 

If your window height is bigger than that, it would display the default white background after the footer (I didn't want to mess with that style). Again not a new issue, just looks a bit weirder now that the footer background is black-ish.

Reply:
oh, I read that as `55px` so it seemed like a big increase.  Changing the page background to black would probably address that but I guess it doesn't matter.

Reply:
I added a background to `html` to match the footer, but kept the `min-height` because I think it's better visually. 

Here is what it would look like without the `min-height`: ![Screenshot 2023-10-02 at 15-54-21 Not allowed](https://github.com/mozilla/addons-server/assets/187006/cce9f76a-673e-48d7-9013-6880df32cbbd)

(Without the matching background color you'd also see white instead of black after the footer's content if the `min-height` is absent or not enough compared to the window height)